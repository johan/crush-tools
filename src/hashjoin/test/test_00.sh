test_number=00
description="two-key join by label"

infile="$test_dir/input_header.log"
dimfile="$test_dir/dimension_header.log"
outfile="$test_dir/test_$test_number.actual"
expected="$test_dir/test_$test_number.expected"

$bin -K 'Field-0,Field-1' -J 'Field-2,Field-3' -f $dimfile $infile \
  > "$outfile"

if [ $? -ne 0 ] ||
   [ "`diff -q $outfile $expected`" ]; then
  test_status $test_number 1 "$description" FAIL
else
  test_status $test_number 1 "$description" PASS
  rm "$outfile"
fi
