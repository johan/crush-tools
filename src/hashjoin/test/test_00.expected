Field-0,Field-1,Something-Else,Field-2,Field-3
1,2,wee,3,4
5,6,w00t,7,8
7,8,nope,,
